Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -273,10 +334,40 @@ class VDPortGroup : BaseDSC {
         $distributedPortGroupParams = $this.GetDistributedPortGroupParams()
 
         try {
+            Write-VerboseLog -Message $this.ModifyVDPortGroupMessage -Arguments @($distributedPortGroup.Name)
             $distributedPortGroup | Set-VDPortgroup @distributedPortGroupParams
         }
         catch {
-            throw "Cannot update Distributed Port Group $($this.Name). For more information: $($_.Exception.Message)"
+            throw ($this.CouldNotModifyVDPortGroupMessage -f $distributedPortGroup.Name, $_.Exception.Message)
+        }
+    }
+
+    <#
+    .DESCRIPTION
+
+    Modifies the VLAN configuration of the specified Distributed Port Group.
+    #>
+    [void] UpdateDistributedPortGroupVlanConfiguration($distributedPortGroup) {
+        $setVDVlanConfigurationParams = @{
+            VDPortgroup = $distributedPortGroup
+            Confirm = $false
+            ErrorAction = 'Stop'
+            Verbose = $false
+        }
+
+        if ($this.VLanId -eq 0) {
+            $setVDVlanConfigurationParams.DisableVlan = $true
+        }
+        else {
+            $setVDVlanConfigurationParams.VlanId = $this.VLanId
+        }
+
+        try {
+            Write-VerboseLog -Message $this.ModifyVDPortGroupVlanConfigurationMessage -Arguments @($distributedPortGroup.Name, $this.VLanId)
```

Comment:
Here as well **Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.